cypress/integration/errorPageArticle.js

```diff
@@ -0,0 +1,73 @@
+import {
+  errorMessage,
+  errorPageInlineLink,
+  errorTitle,
+  hasHtmlLangDirAttributes,
+} from '../support/bodyTestHelper';
+import testData from '../../src/app/lib/config/services';
+import services from '../support/config/services';
+
+const nonExistentAsset = 'cxvxrj8tvppo';
+const serviceHasNonExistentArticle = service =>
+  services[service].pageTypes.nonExistentarticle !== undefined;
+
+// These must only ever be run locally as otherwise you're testing
+// the mozart page not the response from this application.
+Object.keys(services)
+  .filter(serviceHasNonExistentArticle)
+  .forEach(service => {
+    describe('Test we get a 404', () => {
+      it('should return a 404 error code', () => {
+        cy.request({
```

Comment:
you've used testResponseCodeAndType below, this isn't testing anything else is it?

Reply:
This test seems like a duplicate.